cmd/net_test.go

```diff
@@ -291,10 +291,9 @@ func TestSameLocalAddrs(t *testing.T) {
 		expectedErr error
 	}{
 		{"", "", false, errors.New("unable to process empty address")},
-		{":9000", ":9000", true, nil},
-		{"localhost:9000", ":9000", true, nil},
-		{"localhost:9000", "http://localhost:9000", true, nil},
-		{"8.8.8.8:9000", "http://localhost:9000", false, nil},
+		{"http://localhost:9000", ":9000", true, nil},
+		{"http://localhost:9000", "http://localhost:9000", true, nil},
+		{"http://8.8.8.8:9000", "http://localhost:9000", false, nil},
```

Comment:
> I think you need to keep previous cases in addition.

Done now.. for that I made some few changes, can you review again? @balamurugana